src/components/authorization/policies/by-role/fundraising.policy.ts

```diff
@@ -2,46 +2,29 @@ import { member, Policy, Role, sensMediumOrLower } from '../util';
 
 // NOTE: There could be other permissions for this role from other policies
 @Policy(Role.Fundraising, (r) => [
-  r.Budget.read,
-  r.BudgetRecord.read,
-  r.Ceremony.read,
-  r.Directory.read,
-  r.Education.read,
   r.EthnologueLanguage.whenAny(member, sensMediumOrLower).read,
-  r.FieldRegion.read,
-  r.FieldZone.read,
-  r.File.read,
-  r.FileVersion.read,
-  r.FundingAccount.read,
-  r.Engagement.read,
   r.Language.read.specifically((p) => [
     p.registryOfDialectsCode.whenAny(member, sensMediumOrLower).read,
     p.signLanguageCode.whenAny(member, sensMediumOrLower).read,
     p.locations.whenAny(member, sensMediumOrLower).read,
   ]),
-  r.Organization.whenAny(member, sensMediumOrLower).read.specifically((p) => [
+  r.Organization.whenAny(sensMediumOrLower).read.specifically((p) => [
+    //TODO - this none can be removed when policy executor is refactored to check for address.none in parent policy
     p.address.none,
```

Comment:
This is tricky. With more thought, I don't think this should just be allowed without noting something in the other policy. Otherwise we break the guideline of not knowing if X line will actually apply and not be overridden by another.

Currently if not specified all props fallback to resource level perms. I think this should still be the default as it's probably the more common case. So we need a flag to turn off this default.
At first I thought of something like
```ts
r.Organization.noProps
```
which would set all properties to `none`. 
The same as
```ts
r.Organization.specifically(p => p.many(...keys(p)).none
```
But I don't think this is entirely what we want, as we still want to fallback to resource level if no other policy specifies any specific prop permissions.

So instead we need a flag and an update to the executor logic.
```ts
r.Organization.deferProps
```
And Executor does this:
1. Policy A - checks for prop, sees this skips
2. Policy B - checks for prop. If exists, uses that; else, continue
3. Policy A - uses resource level

Thoughts? Thoughts on `deferProps` as name of flag?

Reply:
Upon further reflection, I think this is the edge case. Instead we should be explicit when we want all resource level permission to apply to all properties. The flag could be something like `.andAllProps`?

Reply:
Handled via [2562](https://github.com/SeedCompany/cord-api-v3/pull/2562)